rasa/core/channels/botframework.py

```diff
@@ -232,8 +232,18 @@ async def webhook(request: Request) -> HTTPResponse:
                         postdata["serviceUrl"],
                     )
 
+                    text = ""
+                    if postdata.get("text"):
```

Comment:
would be great to have a link to the documentation of MS botframework that describes the request format, for reference

Reply:
Hey @m-vdb sure the doc is here : https://docs.microsoft.com/en-us/azure/bot-service/rest-api/bot-framework-rest-connector-api-reference?view=azure-bot-service-4.0#activity-object

It's not completely clear that `text` or `value` can be empty and will be empty depending on what you are sending. That wasn't clear to us in the first implementation of this connector :)

Reply:
sounds good 👍🏻 can you add that to the docstring of the function? it'll be easier for other contributors to dive into the code next time we want to implement a change